website/docs/reference/warehouse-setups/infer-setup.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Infer setup"
+id: "infer-setup"
+meta:
+  maintained_by: Infer
+  authors: Erik Mathiesen-Dreyfus, Ryan Garland
+  github_repo: 'inferlabs/dbt-infer'
+  pypi_package: 'dbt-infer'
+  min_core_version: 'v1.2.0'
+  cloud_support: Not Supported
+  slack_channel_name: n/a
+  slack_channel_link: 
+  platform_name: 'Infer'
+  config_page: 'no-configs'
+  min_supported_version: n/a
+---
+
+<h2> Overview of {frontMatter.meta.pypi_package} </h2>
+
+<ul>
+    <li><strong>Maintained by</strong>: {frontMatter.meta.maintained_by}</li>
+    <li><strong>Authors</strong>: {frontMatter.meta.authors}</li>
+    <li><strong>GitHub repo</strong>: <a href={`https://github.com/${frontMatter.meta.github_repo}`}>{frontMatter.meta.github_repo}</a><a href={`https://github.com/${frontMatter.meta.github_repo}`}><img src={`https://img.shields.io/github/stars/${frontMatter.meta.github_repo}?style=for-the-badge`}/></a></li>
+    <li><strong>PyPI package</strong>: <code>{frontMatter.meta.pypi_package}</code> <a href={`https://badge.fury.io/py/${frontMatter.meta.pypi_package}`}><img src={`https://badge.fury.io/py/${frontMatter.meta.pypi_package}.svg`}/></a></li>
+    <li><strong>Slack channel</strong>: <a href={frontMatter.meta.slack_channel_link}>{frontMatter.meta.slack_channel_name}</a></li>
+    <li><strong>Supported dbt Core version</strong>: {frontMatter.meta.min_core_version} and newer</li>
+    <li><strong>dbt Cloud support</strong>: {frontMatter.meta.cloud_support}</li>
+    <li><strong>Minimum data platform version</strong>: {frontMatter.meta.min_supported_version}</li>
+    </ul>
+
+
+<h2> Installing {frontMatter.meta.pypi_package} </h2>
+
+pip is the easiest way to install the adapter:
+
+<code>pip install {frontMatter.meta.pypi_package}</code>
+
+<p>Installing <code>{frontMatter.meta.pypi_package}</code> will also install <code>dbt-core</code> and any other dependencies.</p>
+
+<h2> Configuring {frontMatter.meta.pypi_package} </h2>
+
+<p>For further info, refer to the GitHub repository: <a href={`https://github.com/${frontMatter.meta.github_repo}`}>{frontMatter.meta.github_repo}</a></p>
+
+
+## Connecting to Infer with **dbt-infer**
+
+Infer allows you to perform advanced ML Analytics within SQL as if native to your data warehouse.
+To do this Infer uses a variant called <a href="https://docs.getinfer.io/docs/reference">SQL-inf</a> 
+which defines as set of primitive ML commands from which 
+you can build advanced analysis for any business use case.
+Learn more about SQL-inf and Infer <a href="https://docs.getinfer.io/">here</a>.
```

Comment:
we've got an opinionated way of linking, check out the [Links section of our dbt Style Guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#Links)

I see other links/anchors, I hope you can also catch those

Reply:
Thanks! Fixed all the links now to follow the guide-lines